server/api/controllers/article.js

```diff
@@ -579,4 +578,71 @@ export const getArticle = async (req, res) => {
       message: 'Internal server error'
     });
   }
+};
+  /**
+ * @export
+ * @function updateArticle
+ * @param {Object} req - request received
+ * @param {Object} res - response object
+ * @returns {Object} JSON object (JSend format)
+ */
+export const updateArticle = async (req, res) => {
+  try {
+    const {
+      body,
+      params: {
+        articleId
+      },
+      user: {
+        id: userId
+      }
+    } = req;
+
+    const foundArticle = await Article.findOne({
```

Comment:
@oyedejipeace lines `601` to `614` becomes redundant due to `615`. Please remove. Change the variable name on line ` 615` to `article`